### PR TITLE
docs: c and c++ apis docs improved

### PR DESCRIPTION
**Pull request by mgrudzinska**

### CODE REVIEW

**Review by mihashco — APPROVED** [review]

File: inc/thorvg.h [hunk 1]
```diff
@@ -1144,8 +1145,8 @@ class TVG_EXPORT Scene final : public Paint
     /**
      * @brief Passes drawing elements to the Scene using Paint objects.
      *
-     * Only pushed paints in the scene will be drawing targets.
-     * They are retained by the scene until you call Scene::clear().
+     * Only the paints pushed into the scene will be drawing targets.
+     * The paints are retained by the scene until the Scene::clear() is called.
```
Comment (hermet) [inline inc/thorvg.h:1149]: hmm... is it proper to use "the" for the method()?
https://www.cairographics.org/manual/cairo-cairo-scaled-font-t.html

**Review by hermet — COMMENTED** [review]
@mgrudzinska could you please check a comment?

File: inc/thorvg.h (hunk 1 above)
Comment (mgrudzinska) [reply inc/thorvg.h:1149]: well... it sounds ok here. I can remove if you wish, it'll be still correct. 
I'm not a native :)
I changed here "Only the paints pushed into the scene will be drawing targets." -> "Only the paints pushed into the scene will be the drawn targets."

Comment (mgrudzinska) [reply inc/thorvg.h:1149]: I also changed "in the advance" -> "in advance"

Comment (hermet) [reply inc/thorvg.h:1149]: Question: Are they wrong sentence? 

Cairo: "as they would be drawn by cairo_show_glyphs() "
Direct3D: "A handle to a DLL that implements a software rasterizer. If DriverType is D3D_DRIVER_TYPE_SOFTWARE, Software must not be NULL. Get the handle by calling LoadLibrary, LoadLibraryEx , or GetModuleHandle."

Simpler code and docs are better for users. if it doesn't violate the (programming) grammar.

Comment (mgrudzinska) [reply inc/thorvg.h]: as I said, I'm not a native. fixed

**Pull request merged**